<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaisseDialog</class>
 <widget class="QDialog" name="CaisseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelId">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrenom">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
